appside/render_body_frames.js

```diff
@@ -1,9 +1,36 @@
 "use strict";
```

Comment:
Suggestion for all js files: include a small description of the purpose of file at start of document.
I had a hard time figuring out what render_body_frames did.